From cd7cae4a7b112574207750afd759ab7fec7be5cb Mon Sep 17 00:00:00 2001
From: "flinux@vongellhorn.ch" <flinux@vongellhorn.ch>
Date: Thu, 20 Jun 2019 14:41:30 +0200
Subject: [PATCH 1/1] Solus tweaks to the default config

 - Remove toolbox icon from desktop
 - Decrease desktop icon sizes
 - Add shortcuts for Software Center in menu
 - Set default Monospace font size to 10
 - move showdesktop in front of systemtray
---
 applets/kicker/package/contents/config/main.xml               | 2 +-
 applets/kickoff/package/contents/config/main.xml              | 4 ++--
 containments/desktop/package/contents/config/main.xml         | 4 ++--
 kcms/fonts/fonts.cpp                                          | 2 +-
 .../org.kde.plasma.desktop.defaultPanel/contents/layout.js    | 2 +-
 5 files changed, 7 insertions(+), 7 deletions(-)

diff --git a/applets/kicker/package/contents/config/main.xml b/applets/kicker/package/contents/config/main.xml
index cddcad4..eb20d16 100644
--- a/applets/kicker/package/contents/config/main.xml
+++ b/applets/kicker/package/contents/config/main.xml
@@ -38,7 +38,7 @@
     </entry>
     <entry name="favoriteApps" type="StringList">
       <label>List of general favorites. Supported values are menu id's (usually .desktop file names), special URLs that expand into default applications (e.g. preferred://browser), document URLs and KPeople contact URIs.</label>
-      <default>preferred://browser,kontact.desktop,systemsettings.desktop,org.kde.dolphin.desktop,ktp-contactlist.desktop,org.kde.kate.desktop,org.kde.discover</default>
+      <default>preferred://browser,kontact.desktop,systemsettings.desktop,org.kde.dolphin.desktop,ktp-contactlist.desktop,org.kde.kate.desktop,solus-sc.desktop</default>
     </entry>
     <entry name="favoriteSystemActions" type="StringList">
       <label>List of system action favorites.</label>
diff --git a/applets/kickoff/package/contents/config/main.xml b/applets/kickoff/package/contents/config/main.xml
index be031dc..fba96eb 100644
--- a/applets/kickoff/package/contents/config/main.xml
+++ b/applets/kickoff/package/contents/config/main.xml
@@ -20,7 +20,7 @@
     </entry>
     <entry name="favorites" type="StringList">
       <label>List of general favorites. Supported values are menu id's (usually .desktop file names), special URLs that expand into default applications (e.g. preferred://browser), document URLs and KPeople contact URIs.</label>
-      <default>preferred://browser,kontact.desktop,systemsettings.desktop,org.kde.dolphin.desktop,ktp-contactlist.desktop,org.kde.kate.desktop,org.kde.discover.desktop</default>
+      <default>preferred://browser,kontact.desktop,systemsettings.desktop,org.kde.dolphin.desktop,ktp-contactlist.desktop,org.kde.kate.desktop,solus-sc.desktop</default>
     </entry>
     <entry name="favoritesPortedToKAstats" type="Bool">
       <label>Are the favorites ported to use KActivitiesStats to allow per-activity favorites</label>
@@ -28,7 +28,7 @@
     </entry>
     <entry name="systemApplications" type="StringList">
       <label>List of applications at the top of the "Computer" tab.</label>
-      <default>systemsettings.desktop,org.kde.kinfocenter.desktop,org.kde.discover.desktop</default>
+      <default>systemsettings.desktop,org.kde.kinfocenter.desktop,solus-sc.desktop</default>
     </entry>
     <entry name="useExtraRunners" type="Bool">
       <label>Whether to use additional KRunner plugins to produce results in the search.</label>
diff --git a/containments/desktop/package/contents/config/main.xml b/containments/desktop/package/contents/config/main.xml
index 43a4935..78fb7ac 100644
--- a/containments/desktop/package/contents/config/main.xml
+++ b/containments/desktop/package/contents/config/main.xml
@@ -100,7 +100,7 @@
     </entry>
     <entry name="iconSize" type="Int">
       <label>The icon size to use for Folder View icons. One of the list of Plasma icon sizes.</label>
-      <default>4</default>
+      <default>3</default>
     </entry>
     <entry name="labelWidth" type="Int">
       <label>The width of the grid cells when using icon view. 0 = Narrow, 1 = Medium, 2 = Wide</label>
@@ -138,7 +138,7 @@
     </entry>
     <entry name="showToolbox" type="Bool">
       <label>Whether to show the Desktop Toolbox. Only used when serving as containment.</label>
-      <default>true</default>
+      <default>false</default>
     </entry>
     <entry name="pressToMove" type="Bool">
       <label>Whether to activate widget handles and widget move mode by long press. Only used when serving as containment.</label>
diff --git a/kcms/fonts/fonts.cpp b/kcms/fonts/fonts.cpp
index faed0a4..b705f04 100644
--- a/kcms/fonts/fonts.cpp
+++ b/kcms/fonts/fonts.cpp
@@ -476,7 +476,7 @@ void KFonts::defaults()
 #else
     setGeneralFont(QFont("Noto Sans", 10));
     setMenuFont(QFont("Noto Sans", 10));
-    setFixedWidthFont(QFont("Hack", 9));
+    setFixedWidthFont(QFont("Hack", 10));
     setToolbarFont(QFont("Noto Sans", 10));
     setSmallFont(QFont("Noto Sans", 8));
     setWindowTitleFont(QFont("Noto Sans", 10));
diff --git a/layout-templates/org.kde.plasma.desktop.defaultPanel/contents/layout.js b/layout-templates/org.kde.plasma.desktop.defaultPanel/contents/layout.js
index 048671e..e52d6d9 100644
--- a/layout-templates/org.kde.plasma.desktop.defaultPanel/contents/layout.js
+++ b/layout-templates/org.kde.plasma.desktop.defaultPanel/contents/layout.js
@@ -78,7 +78,7 @@ if (langIds.indexOf(languageId) != -1) {
     panel.addWidget("org.kde.plasma.kimpanel");
 }
 
+panel.addWidget("org.kde.plasma.showdesktop")
 panel.addWidget("org.kde.plasma.systemtray")
 panel.addWidget("org.kde.plasma.digitalclock")
-panel.addWidget("org.kde.plasma.showdesktop")
 
-- 
2.22.0

